src/rdkafka_admin.c

```diff
@@ -5244,6 +5245,18 @@ void rd_kafka_AlterConsumerGroupOffsets(
                 }
         }
 
+        /* Copy offsets list for checking duplicated */
+        copied_offsets =
+            rd_kafka_topic_partition_list_copy(alter_grpoffsets[0]->partitions);
+        if (rd_kafka_topic_partition_list_has_duplicates(
+                copied_offsets, rd_false /* ignore partition */)) {
```

Comment:
Shouldn't the comment be - "don't ignore partition"? Its confusing, better to remove the comment.

we are making `ignore_partition = rd_false` here.

Reply:
I put `/*check partition*/` like in the other place